app/models/performer.rb

```diff
@@ -1,2 +1,6 @@
 class Performer < ActiveRecord::Base
+  has_many :films, through: :films_performers
```

Comment:
same here, you'll need a class `FilmsPerformer` and in that you'll need two belongs to associations in order to get this has_many/through association to work